_help/index.md

```diff
@@ -31,12 +33,13 @@ links:
       <ul class="pl0">
       {% for subpage in page.links %}
         {% assign subpage_slug = subpage.name | slugify %}
-        {% assign first_subpage = site.help | where_exp: "item", "item.url contains subpage_slug" | where: 'order', 1 | first %}
+        {% assign lang_path = "/" | append: site.lang | append: "/" %}
+        {% assign first_subpage = site.help | where_exp: "item", "item.url contains lang_path" | where_exp: "item", "item.url contains subpage_slug" | where: 'order', 1 | first %}
```

Comment:
So the other option was to move all the help articles into either the yaml file or a separate translations folder. 

The way it is currently setup, allows us to just put the different help files in sub folders and have it still function correctly without changing the other templates.